src/main/java/seedu/duke/command/ListIngredientCommand.java

```diff
@@ -7,6 +7,8 @@
  * Lists all ingredients used in the selected dish to the user.
  */
 public class ListIngredientCommand extends Command{
+    public static final String COMMAND_WORD = "list ingredients";
```

Comment:
change the command word to match what we stated in user guide
```suggestion
    public static final String COMMAND_WORD = "list_ingredients";
```